dashboard/test/ui/runner.rb

```diff
@@ -260,8 +261,10 @@ def run_tests(arguments)
   File.open(status_page_filename, 'w') do |file|
     file.write haml_engine.render(Object.new, {
       api_origin: CDO.studio_url('', scheme),
+      s3_bucket: S3_LOGS_BUCKET,
+      s3_prefix: S3_LOGS_PREFIX,
       type: test_type,
-      git_branch: git_branch,
+      git_branch: GIT_BRANCH,
```

Comment:
I still pass the branch name down to the template, but it's only used for display purposes now - not for generating URLs to fetch test status.